source/cloud/azure/azureml.md

```diff
@@ -232,7 +233,7 @@ returned_job = ml_client.jobs.create_or_update(command_job)
 
 After creating the job, go to [the "Experiments" page](https://ml.azure.com/experiments) to view logs, metrics, and outputs.
 
-Next, try performing a sweep over a set of hyperparameters.
+Next, perform a sweep over a set of hyperparameters. This will take (insert time?)
```

Comment:
It's unclear how long things take, because in the notebook everything runs, but how are we sure things are running? What are we expecting here as output? 

Does this take a long time to run? Maybe we should disclaim this, because if this is your first time running it, it's unclear.

Reply:
I went to the experiments page, but it was in "preparing" status for a long time, so I wasn't sure what I was looking for in there, and it seemed too long so I ended up cutting the experiment. 

Similar to the sweep step, I was not seeing anything so I was not sure what to expect. I think the whole job takes a long time, maybe @jameslamb knows, since you've run this before.

Reply:
Not sure how long "long time" is. When I ran this in #484, I started seeing results on that page after about 20 minutes. And then they updated live as more experiments were run.

If I recall correctly, the entire job with the configuration in these docs took on the order of low-single-digit hours to complete.